docs/developer/system-updates.md

```diff
@@ -0,0 +1,176 @@
+# System Updates: Publishing and Targeting Updates
+
+## Where to Add Updates
+
+All system updates are defined in a single shared UpdateRegistry widget file. To publish or modify updates, edit the following file:
+
+[instances/widgets.treasury-factory.near/widget/pages/settings/system-updates/UpdateRegistry.jsx](../../instances/widgets.treasury-factory.near/widget/pages/settings/system-updates/UpdateRegistry.jsx)
+
+> **Note:** All instances share this UpdateRegistry file. Updates published here will be visible to all instances unless using the `instances` property for targeting specific instances as described below.
+
+The system update functionality allows developers to publish updates that has to be applied and approved manually by treasury administrators (such as contract upgrades, policy changes, or instance app widget changes) to specific treasury instances or to all. This is managed through the `UpdateRegistry` widget and can be tested or previewed using the Playwright test suite.
+
+## How Updates Are Defined
+
+Updates are defined as JavaScript objects in the `UpdateRegistry` widget. Each update record can include the following fields:
+
+- `id` (number): Unique identifier for the update.
+- `createdDate` (string): Date of the update in `YYYY-MM-DD` format.
+- `version` (string): Version or label for the update (optional).
+- `type` (string): Type/category of the update (e.g., "DAO contract").
+- `summary` (string): Short summary of the update.
+- `details` (string): Detailed description (optional).
+- `instances` (array of strings): (Optional) List of instance account IDs that should receive this update. If omitted, the update is shown to all instances.
+- `votingRequired` (boolean): Whether a vote is required for this update.
+
```

Comment:
I think we should change it a bit, because here vote means the dao vote right? and it will be a dao configuration vote (approved by admin role)

```suggestion
- `votingRequired` (boolean): Whether a DAO vote by admin is required after the update request is created.
```